tools/quantize/block_quantize.py

```diff
@@ -0,0 +1,432 @@
+import argparse
+import os
+from dataclasses import dataclass, field
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import onnx
+from onnx import helper
+
+BITS_TO_NUMPY_TYPE = {8: np.uint8, 16: np.uint16}
+
+CONVOLUTION = "Conv"
+
+KERNEL_SHAPE = "kernel_shape"
+STRIDE = "strides"
+PADS = "pads"
+DILATIONS = "dilations"
+GROUP = "group"
```

Comment:
Could you pack them in a dictionary so that it can be more generic. Related code should be changed accordingly for generalization so that we can easily apply the similar quantization to `MatMul` or `Gemm`.

Reply:
Those parameters were actually not used. I refactored the code to make it more general.